simplecpp.cpp

```diff
@@ -1737,16 +1739,10 @@ namespace {
         }
     }
 
+    const char * const altopData[] = {"and","or","bitand","bitor","not","not_eq","xor"};
```

Comment:
Maybe this should have been addressed by astyle. I am having problems running it when developing on cygwin. Usually the version is not the required one, or I've seen astyle crashing on cygwin also.